governance-exercises/poll-ex1-response.md

```diff
@@ -0,0 +1,46 @@
+
+- After determining the contents of the poll I would choose from the templates folder the most suitable templates to slot this poll content into, which in this case are 'Adjust $protocol_name D3M Parameters - $date_MONTH_DD,_YYYY.md' and 'Onboard $token to Spark Protocol - $date_MONTH_DD,_YYYY.md'.
+- Next, I would publish a response in the forum thread informing the proposers and community of the decision to submit two polls, with rationale included. *See forum response below*
+- I would then begin to draft the polls using the poll templates as noted above. The polls will include a summary of the proposal, voting choices, and links to relevant resources for voters to make informed decisions. I will review the poll drafts against the 'poll creation checklist' to ensure they meet all criteria. Additionally, I will carefully check for any spelling or grammatical errors and make sure the language used is easy to understand.
+- I would then create a draft PR to the repo and request reviews of the polls content from other scope facilitators, the proposers and Bob
+- Using any feedback and notes from reviewers, I will finalise the polls and review them against the 'poll submission checklist', adding all relevant tags based on the potential impact of the proposed changes and their appropriate categories. In this case, relevant tags would include "high-impact", collateral-onboard" and "D3M" before making a PR to push final changes to the main repo branch.
+- Once the polls are active I would start comms across all relevant channels including governance channels in the MakerDAO Discord, and within the original forum thread itself, for the sake of visibility and awareness. I would continue to monitor these channels and respond to any questions and concerns.
+
+## 2: Draft any forum responses required or implied by your course of action.
+
+## Forum Response
+
+I'm glad to see the continued success and growth of the Spork protocol, and I appreciate the detailed proposals from both @Alice and @Faraz, as well as @Bob's approval for a governance poll.
+
+After reviewing the recommendations and advisories regarding the requested changes, I have decided to split the proposal into two separate polls. The proposed changes, though closely linked, are quite distinct and have a significant impact, so this approach will allow voters to consider and vote on each matter independently. The two polls will be presented during the next weekly governance cycle on **Monday, July 24th**:
+
+  ### Adjust Spork Protocol D3M Parameters and DAI Interest Rate - July 24, 2023:
+  - Increase Spork D3M Debt Ceiling by 50 million from 50 million to 100 million DAI.
+  - Increase Spork D3M Target Available Debt (gap) by 10 million from 5 million to 15 million.
+  - Adjust Spork Protocol DAI Interest Rate Strategy to increase the DAI deposit rate by 0.5% from 2% to 2.5%
+  - Increase the Ceiling Increase Cooldown from 28,800 seconds (8 hrs) to 86,400 seconds (24 hrs)
```

Comment:
Generally speaking, prefer human readable values. There is not a huge amount of value of including the times in seconds, for example.